build.sbt

```diff
@@ -751,7 +752,17 @@ lazy val yaml = (project in file("lib/java/yaml"))
     frgaalJavaCompilerSetting,
     version := "0.1",
     libraryDependencies ++= Seq(
-      "io.circe" %% "circe-yaml" % circeYamlVersion % "provided"
+      "org.yaml" % "snakeyaml" % snakeyamlVersion % "provided"
+    )
+  )
+
+lazy val `scala-yaml` = (project in file("lib/scala/yaml"))
+  .enablePlugins(JPMSPlugin)
```

Comment:
You can remove this line. Using `JPMSPlugin` is not necessary unless you want to create a module from the project.